plaso/parsers/mobile_installation.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+"""Parser for iOS mobile installation logs files obtained from iOS device.
+Based on the research from Alex Brignoni."""
+
+import pyparsing
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import errors
+from plaso.lib import definitions
+from plaso.parsers import manager
+from plaso.parsers import text_parser
+
+
+class MobileInstallationEventData(events.EventData):
+  """Mobile Installation log event data
+  Attributes:
```

Comment:
Please add a description for all the attributes and sort the attributes alphabetically as per the style guide:
https://github.com/log2timeline/l2tdocs/blob/main/process/Style-guide.md